src/main/resources/org/jenkinsci/plugins/github_branch_source/GitHubAppCredentials/config.jelly

```diff
@@ -0,0 +1,30 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form" xmlns:st="jelly:stapler">
+  <st:include page="id-and-description" class="${descriptor.clazz}"/>
+  <f:entry title="${%App ID}" field="appID">
+    <f:textbox/>
+  </f:entry>
+  <j:choose>
+    <j:when test="${descriptor.apiUriSelectable}">
+      <f:entry title="${%API endpoint}" field="apiUri">
+        <f:select/>
+      </f:entry>
+    </j:when>
+    <j:otherwise>
+      <f:invisibleEntry>
+        <f:select field="apiUri"/>
+      </f:invisibleEntry>
+    </j:otherwise>
+  </j:choose>
+
+  <f:entry title="${%Key}" field="privateKey">
+    <!-- TODO: after upgrading jenkins.version >= 2.171, remove xmlns below and use f:secretTextArea
```

Comment:
Maybe it is time to just bump the dependency. Ni strong opinion

Reply:
Happy to do if maintainers here say so.

Reply:
I just recently bumped to 2.150.3...
https://github.com/jenkinsci/github-branch-source-plugin/blame/master/pom.xml#L28

How important is this?

Reply:
I don’t think it matters, there’s a lib dependency that is allowing us to use this tag lib while on a lower version of core than its released in.

Reply:
+1, no need to bump here